crates/pop-cli/src/commands/up/parachain.rs

```diff
@@ -182,6 +182,7 @@ async fn generate_spec_files(
 		id: Some(id),
 		genesis_code: true,
 		genesis_state: true,
+		deterministic: true,
```

Comment:
Why is this true? What if the user doesnt have Docker/podman?

Reply:
I've removed this https://github.com/r0gue-io/pop-cli/pull/425/commits/bb3bd7c1fe2ccfafd5a7ad3453746d2546fa0b52 and updated the user prompt with the improved message you suggested below.